### PR TITLE
create attend employee

### PR DESCRIPTION
**Pull request by thiennguyen96841996**

### CODE REVIEW

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
comment

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

File: app/Http/Controllers/Employee/AttendController.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $attends = Working::where('user_id', Auth::user()->id)->get();
+        $stastics = working::where('user_id',Auth::user()->id)->whereMonth('date', Carbon::now()->format('m'))->whereYear('date', Carbon::now()->format('Y'))->get();
+        $lates = working::where('user_id',Auth::user()->id)->whereMonth('date', Carbon::now()->format('m'))->whereYear('date', Carbon::now()->format('Y'))->where('status', 0)->get();
+
+        return view("employees.attend.index", compact('attends', 'stastics', 'lates'));
```
Comment (BunnyPi04) [inline app/Http/Controllers/Employee/AttendController.php]: sử dụng nháy đơn

File: app/Http/Controllers/Employee/AttendController.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        $attendsions = new working;
+        $attendsions->date = $timeNow;
+        $attendsions->month = $timeNow->month;
+        $attendsions->time = $time;
+        if($timeNow->hour <= 7 && $timeNow->minute <= 0) {
+            $attendsions->status = 1;
+        }
+        $attendsions->status = 0;
+        $attendsions->user_id = Auth::user()->id;
+        $checkExitDate = working::where('user_id', Auth::user()->id)->orderBy('date','desc')->value('date');
```
Comment (BunnyPi04) [inline app/Http/Controllers/Employee/AttendController.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Pull request merged**